node/loop.go

```diff
@@ -216,20 +207,13 @@ func (t *loopingTransport) loop() {
 					continue
 				}
 
-				t.subscriptionsMu.RLock()
-				if subscription, ok := t.subscriptions[sp.Subscription]; ok {
-					go func(n *jsonrpc.Notification) {
-						_copy := *n
-						select {
-						case <-ctx.Done():
-							return
-						case subscription.ch <- &_copy:
-							return
-						}
-
-					}(msg)
-				}
-				t.subscriptionsMu.RUnlock()
+				func(n jsonrpc.Notification) {
```

Comment:
Hmm: this currently no longer spawns a goroutine, which means that a slow subscriber channel consumer can probably block the loop, so I think we should re-introduce the goroutine here.

Reply:
Ok 5b23a64 re-introduces the `go` so that this is still dispatched asynchronously.